dietpi/dietpi-software

```diff
@@ -12376,6 +12406,55 @@ _EOF_
 
 		fi
 
+    software_id=181 # PaperMC
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+      
+      Create_User -d /opt/papermc papermc # This creates a system user named papermc with /opt/papermc as its home dir.
+      # Bedrock compatibility
+      G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+      if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate plugins for compatibility with Bedrock Edition?
+      \n Note that this may be buggy."; then
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar' /opt/papermc/plugins/Geyser-Spigot.jar
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar' /opt/papermc/plugins/floodgate-bukkit.jar
+      fi
+      Download_Install https://github.com/Tiiffi/mcrcon/archive/v0.7.1.tar.gz
+      G_EXEC gcc mcrcon-0.7.1/mcrcon.c -o /opt/papermc/mcrcon
+      G_EXEC rm -R mcrcon-0.7.1
+      G_EXEC strip --remove-section=.comment --remove-section=.note /opt/papermc/mcrcon # Reduce file size
+      G_EXEC chown -R papermc:papermc /opt/papermc
+      G_EXEC cd /opt/papermc
+      G_EXEC_OUTPUT=1 G_EXEC java -jar /opt/papermc/paperclip.jar nogui
+      G_CONFIG_INJECT 'eula=' 'eula=true' /opt/papermc/eula.txt
```

Comment:
```
[ INFO ] DietPi-Software | java -jar /opt/papermc/paperclip.jar nogui, please wait...
Downloading vanilla jar...
Patching vanilla jar...
System Info: Java 11 (OpenJDK 64-Bit Server VM 11.0.9+11-post-Debian-1deb10u1) Host: Linux 4.19.0-10-amd64 (amd64)
Loading libraries, please wait...
[12:52:35 ERROR]: Failed to load properties from file: server.properties
[12:52:36 WARN]: Failed to load eula.txt
[12:52:36 INFO]: You need to agree to the EULA in order to run the server. Go to eula.txt for more info.
[  OK  ] DietPi-Software | java -jar /opt/papermc/paperclip.jar nogui
[  OK  ] DietPi-Software | Setting in /opt/papermc/eula.txt adjusted: eula=true
```
Let's do `echo 'eula=true' > /opt/papermc/eula.txt` right after EULA has been accepted in install step. The file does not contain anything else, so that is safe to do.

Reply:
Probably not, it really just downloads the server JAR file.  That can be done on first run.  I'll remove it.

Reply:
That btw also solves a wrong permissions issue due to server start as root which broke at least file logging derived from syslogs:
```
2020-10-26 13:21:03 root@VM-Buster:/opt/papermc# l
total 45M
-rw-r--r-- 1 papermc papermc    2 Oct 26 12:55 banned-ips.json
-rw-r--r-- 1 papermc papermc    2 Oct 26 12:55 banned-players.json
-rw-r--r-- 1 papermc papermc 1.2K Oct 26 12:55 bukkit.yml
drwxr-xr-x 2 root    root    4.0K Oct 26 12:51 cache
-rw-r--r-- 1 papermc papermc  598 Oct 26 12:55 commands.yml
-rw-r--r-- 1 root    root     250 Oct 26 12:52 eula.txt
-rw-r--r-- 1 papermc papermc 2.6K Oct 26 12:53 help.yml
drwxr-xr-x 2 root    root    4.0K Oct 26 12:52 logs
-rwxr-xr-x 1 papermc papermc  19K Oct 26 12:51 mcrcon
-rw-r--r-- 1 papermc papermc    2 Oct 26 12:55 ops.json
-rw-r--r-- 1 papermc papermc 6.5K Oct 26 12:56 paper.yml
-rw-r--r-- 1 papermc papermc  45M Oct 26 12:51 paperclip.jar
-rw-r--r-- 1 papermc papermc    0 Oct 26 12:53 permissions.yml
drwxr-xr-x 5 papermc papermc 4.0K Oct 26 12:53 plugins
-rw-r--r-- 1 root    root    1.1K Oct 26 12:55 server.properties
-rw-r--r-- 1 papermc papermc 4.3K Oct 26 12:56 spigot.yml
-rw-r--r-- 1 papermc papermc    2 Oct 26 12:55 usercache.json
-rw-r--r-- 1 papermc papermc   47 Oct 26 12:53 version_history.json
-rw-r--r-- 1 papermc papermc    2 Oct 26 12:53 whitelist.json
drwxr-xr-x 7 papermc papermc 4.0K Oct 26 13:17 world
drwxr-xr-x 3 papermc papermc 4.0K Oct 26 13:17 world_nether
drwxr-xr-x 3 papermc papermc 4.0K Oct 26 13:17 world_the_end
```